include/seqan3/range/views/minimiser.hpp

```diff
@@ -0,0 +1,555 @@
+    constexpr auto operator()(uint32_t const & num_w_elems) const
+    {
+        return adaptor_from_functor{*this, num_w_elems};
+    }
+
+    /*!\brief                   Call the view's constructor with two arguments: the underlying view and an integer
+     *                          indicating how many elements one window contains.
+     * \param[in] urange        The input range to process. Must model std::ranges::viewable_range and
+     *                          std::ranges::forward_range.
+     * \param[in] num_w_elems   The number of elements in one window.
+     * \returns                 A range of converted elements.
+     */
+    template <std::ranges::range urng_t>
+    constexpr auto operator()(urng_t && urange, uint32_t const & num_w_elems) const
+    {
+        static_assert(std::ranges::viewable_range<urng_t>,
+            "The range parameter to views::minimiser cannot be a temporary of a non-view range.");
+        static_assert(std::ranges::forward_range<urng_t>,
+            "The range parameter to views::minimiser must model std::ranges::forward_range.");
+
+        return minimiser{urange, num_w_elems};
+    }
+};
+//![adaptor_def]
+
+} // namespace seqan3::detail
+
+namespace seqan3::views
+{
+
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief                   Computes minimisers for a range of integral numbers. A minimiser is the smallest value in a
+ *                          window.
+ * \tparam urng_t           The type of the range being processed. See below for requirements. [template parameter is
+ *                          omitted in pipe notation]
+ * \param[in] urange        The range being processed. [parameter is omitted in pipe notation]
+ * \param[in] num_w_elems   The number of elements in one window.
+ * \returns                 A range of std::integral where each value is the minimal value for one window.
+ *                          See below for the properties of the returned range.
+ * \ingroup views
+ *
+ * \details
+ *
+ * A minimiser is the smallest value in a window. For example for the following list of hash values
+ * [28, 100, 9, 23, 4, 1, 72, 37, 8] and 4 elements per window, the minimiser values are [9,4,1].
+ *
+ *
+ * ### View properties
+ *
+ * | Concepts and traits              | `urng_t` (underlying range type)   | `rrng_t` (returned range type)   |
+ * |----------------------------------|:----------------------------------:|:--------------------------------:|
+ * | std::ranges::input_range         | *required*                         | *preserved*                      |
+ * | std::ranges::forward_range       | *required*                         | *preserved*                      |
+ * | std::ranges::bidirectional_range |                                    | *lost*                           |
+ * | std::ranges::random_access_range |                                    | *lost*                           |
```

Comment:
here you even document that random access range is lost but above you tried to implement it?